File: Classes/DDHBaseTableViewCell.m
//  Created by dasdom on 17.07.19.
//  
//

#import "DDHBaseTableViewCell.h"

@implementation DDHBaseTableViewCell

+ (NSString *)identifier {
    return NSStringFromClass(self);
}

- (void)updateWithItem:(id)item {
    NSAssert(false, @"Subclasses need to implement this.");
}

@end

File: Classes/Helper/DDHBaseTableViewCell.swift
//  Created by dasdom on 05.08.19.
//  Copyright © 2021 dasdom. All rights reserved.
//

import UIKit

class DDHBaseTableViewCell<T>: UITableViewCell {
  
  static var identifier: String {
    return NSStringFromClass(self)
  }
  
  func update(with item: T) {
    
  }
}
